frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -666,15 +667,18 @@ describe("scenarios > visualizations > pivot tables", () => {
       display: "line",
     });
 
-    cy.findByText("Visualization").click();
-    sidebar().within(() => {
-      // This part is still failing. Uncomment when fixed.
-      // cy.findByText("Pivot Table")
-      //   .parent()
-      //   .should("have.css", "opacity", "1");
-      cy.icon("pivot_table").click({ force: true });
+    cy.findByText("Settings").click();
+    cy.findByTestId("chart-type-widget").within(() => {
+      cy.icon("chevrondown").click();
     });
 
+    // This part is still failing. Uncomment when fixed.
```

Comment:
Why is this check failing now?

Reply:
The isSensible check fails when looking at pivot tables. I didn't look into it yet, though it's probably worth noting that I did add an `isSeneisble` check to waterfall charts.